### PR TITLE
Enabling tsconfig noUnusedLocals

### PR DESCRIPTION
**Pull request by mikzat**

### CODE REVIEW

File: frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts [hunk 1]
```diff
@@ -210,7 +214,7 @@ export class CastDragAndDrop implements OnInit {
       };
       this.castPositions.push(castPosition);
       for (let dancerIndex = 0; dancerIndex < castPosition.dancerCount;
-          dancerIndex++) {
+           dancerIndex++) {
         castPosition.castRows.push({
           subCastDancers: new Array(this.castCount),
```
Comment (yhedholm) [inline frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts:219]: Is the extra space in front of dancerIndex++) { a typo or purposeful?

File: frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts [hunk 2]
```diff
@@ -219,21 +223,21 @@ export class CastDragAndDrop implements OnInit {
...
+              castPos => castPos.pos.uuid === b.position_uuid).pos.order;
+          return castPositionOrderA < castPositionOrderB ? -1 : 1;
+        });
 
     for (let posIndex = 0; posIndex < this.cast.filled_positions.length;
-        posIndex++) {
+         posIndex++) {
       const filledPos = this.cast.filled_positions[posIndex];
       const uiPos = this.castPositions[posIndex];
       let maxDancerIndex = 0;
```
Comment (yhedholm) [inline frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts:243]: I guess it is purposeful to line up with the prior line's parameters?

**Review by yhedholm — APPROVED** [review]

File: frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts (hunk 1 above)
Comment (mikzat) [reply frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts:219]: Looks like it's lining up with the open parenthesis of the for-loop

File: frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts (hunk 2 above)
Comment (mikzat) [reply frontend/rolecall/src/app/cast/cast-drag-and-drop.component.ts:243]: same

**Pull request closed without merging**